CONTRIBUTING.md

```diff
@@ -0,0 +1,487 @@
+# Contributing to Pulsar
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+The following is a set of guidelines for contributing to Pulsar and its packages, which are hosted in the [Pulsar Organization](https://github.com/pulsar-edit) on GitHub. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+#### Table Of Contents
+
+[Code of Conduct](#code-of-conduct)
+
+[I don't want to read this whole thing, I just have a question!!!](#i-dont-want-to-read-this-whole-thing-i-just-have-a-question)
+
+[What should I know before I get started?](#what-should-i-know-before-i-get-started)
+  * [Pulsar and Packages](#pulsar-and-packages)
+  * [Pulsar Design Decisions](#design-decisions)
+
+[How Can I Contribute?](#how-can-i-contribute)
+  * [Reporting Bugs](#reporting-bugs)
+  * [Suggesting Enhancements](#suggesting-enhancements)
+  * [Your First Code Contribution](#your-first-code-contribution)
+  * [Pull Requests](#pull-requests)
+
+[Styleguides](#styleguides)
+  * [Git Commit Messages](#git-commit-messages)
+  * [JavaScript Styleguide](#javascript-styleguide)
+  * [CoffeeScript Styleguide](#coffeescript-styleguide)
+  * [Specs Styleguide](#specs-styleguide)
+  * [Documentation Styleguide](#documentation-styleguide)
+
+[Additional Notes](#additional-notes)
+  * [Issue and Pull Request Labels](#issue-and-pull-request-labels)
+
+## Code of Conduct
+
+This project and everyone participating in it is governed by the [Pulsar Code of Conduct](CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. Please report unacceptable behavior to the maintainers or GitHub.
+
+## I don't want to read this whole thing I just have a question!!!
+
+> **Note:** [Please don't file an issue to ask a question.](https://github.com/orgs/pulsar-edit/discussions) You'll get faster results by using the resources below.
```

Comment:
Add the permalink to the discord here. https://discord.gg/7aEbB9dGRT

Reply:
Replaced the link here

Reply:
I meant for that to be added in addition tbh to our own discussions listed here since we have that set up org wide

Reply:
Ahh I see what you mean. I mean really the next line there is then linking to both our discussions and the discord server. Maybe we just remove the singular link altogether and that lets the two links directly below stand out a bit more?

Reply:
Alright, its been a while, so I'm gonna go ahead and just remove the singular link listed, and instead let the other two links be more noticeable as the best ways to contact us.